views/templates/admin/configuration.tpl

```diff
@@ -68,8 +69,7 @@
     <div class="mj_landing">
         <div align="center">
             <br />
-            <a href="https://www.mailjet.com/support"
-               target="_blank">{l s='If you have a question or if you have a problem, click here to contact support.' mod='mailjet'}</a>
+            {{l s='If you have a question or if you have a problem, click here to contact support.' mod='mailjet'}|html_entity_decode}
```

Comment:
@Quetzacoalt91 it turns out that without the decoding the text shows as:
```javascript
Should you have any questions or encounter any difficulties, please consult our <a target="_blank" href="https://www.mailjet.com/guides/prestashop-user-guide"><strong>User Guide</strong></a> or contact our technical <a target="_blank" href="https://www.mailjet.com/support/ticket"><strong>Support Team</strong></a>
```
will put back the decoding if you don't mind...